### DISCUSSION TITLE
Is it possible to paste a Word or Excel table in a quarto document?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#6172] [Q&A]
### Description

Is it possible to paste a Word or Excel table in a quarto document?

**Comment by rgaiacs** [comment]
The way that a Word or Excel table is handle will depend of the text editor that you are using. Can you provide more details of your environment?

**Comment by ManuelSpinola** [comment]
Just cut and paste a table from a Word document and paste it in a quarto document.

**Reply by mcanouil** [reply]
This is not a Quarto thing, but you can have extension in VSCode to do that, see <https://marketplace.visualstudio.com/search?term=copy%20excel&target=VSCode&category=All%20categories&sortBy=Relevance>.
But you won't get styling.

PS: we know how to copy/paste, but as said, copy/paste depends on the editor, _i.e._, VSCode for the extensions showed above.

**Comment by baptiste** [comment]
I would personally run

```
quarto pandoc document.docx -t markdown 
```

and you can copy-paste the resulting markdown table in your document.

**Reply by mcanouil** [reply]
Indeed, although it is not a direct copy/paste and does not work for Excel document.

**Comment by ManuelSpinola** [comment]
Thank you very much for the suggestions.

I also found that the RStudio Addin datapasta can help.



El mar, 11 jul 2023 a las 14:40, Mickaël Canouil ***@***.***>)
escribió:

> Indeed, although it is not a direct copy/paste and does not work for Excel
> document.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/6172#discussioncomment-6420373>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB35P2HLQO6VO5OZIKTXPW247ANCNFSM6AAAAAA2GCK3AY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>
